assembl/templates/notifications/html_mail.jinja2

```diff
@@ -79,16 +79,14 @@ assembl_css
                       <table class="twelve columns">
                         <tr>
                           <td class="panel">
-                            <p>{{ gettext("You are receiving this because you are %(notificationConfigLink)s to discussion %(discussionLink)s.  Specifically, you currently receive a notification when:",
-                                    discussionLink=discussionLink,
-                                    notificationConfigLink='<a href="'+notificationConfigUrl+'">'+gettext('subscribed')+'</a>')
+                            <p>{{ gettext("You are receiving this because you are subscribed to discussion %(discussionLink)s.  Specifically, you currently receive a notification when:",
+                                    discussionLink=discussionLink)
                                }}
                             <ul>
-                              
+
                               {% for applicableSubscription in notification.get_applicable_subscriptions() %}
                                 <li>
                                   {{ gettext(applicableSubscription.get_human_readable_description()) }}
-                                  <a href="{{ frontendUrls.getUserNotificationSubscriptionUnsubscribeUrl(applicableSubscription) }}">{{ gettext('Unsubscribe') }}</a>
```

Comment:
@n0izn0iz This is an issue. There are EU laws requiring a link to be able to unsubscribe. Speak with Fred with regards to fixing the underlying issue, which is, we need a functional unsubscribe link that works in V2 interface.
https://litmus.com/blog/gdpr-what-europes-new-privacy-law-means-for-email-marketers